lottie/src/main/java/com/airbnb/lottie/parser/DropShadowEffectParser.java

```diff
@@ -67,10 +67,10 @@ private void maybeParseInnerEffect(JsonReader reader, LottieComposition composit
               direction = AnimatableValueParser.parseFloat(reader, composition, false);
               break;
             case "Distance":
-              distance = AnimatableValueParser.parseFloat(reader, composition);
+              distance = AnimatableValueParser.parseFloat(reader, composition, false);
               break;
             case "Softness":
-              radius = AnimatableValueParser.parseFloat(reader, composition);
+              radius = AnimatableValueParser.parseFloat(reader, composition, false);
```

Comment:
I removed the dp -> px scale factor here since it's not clear to me that the units used by After Effects map to one or the other.

Reply:
Just did some tests and I think you're right here